prive/configs/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "num_train_samples": 100,
+    "num_test_samples": 100,
+    "train_data_size": 5,
+    "synthetic_data_size": 5,
+    "test_as_aux_frac": 0.5,
+    "attacks": [
+        {
+            "name": "Groundhog",
+            "params": {
+                "setrep": "NaiveRep",
+                "classifier": "RandomForest"
+            }
+        }
+    ]
+}
```

Comment:
**Optional** suggestion:

Unless supplied on the command line, look for config files first in the XDG standard places (https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), then in the default `prive/configs` location.

See, eg, https://pypi.org/project/xdg/ (which I have not used)

Reply:
I don't feel strongly for or against this. Is the XDG specification something that would be unique to `prive` or is this a system-wide setting? Happy to make the script look in `prive/configs`.